packages/opds-web-client/src/reducers/loans.ts

```diff
@@ -26,6 +26,16 @@ export default (state: LoansState = initialState, action): LoansState => {
         books: action.books
       });
 
+    case "LOAD_UPDATE_BOOK_DATA":
+      // A book has been updated, so the loans feed is now outdated.
+      // If we remove the loans, the components showing the book that
+      // was updated can use the data from the book update request 
+      // until the next LOAD_LOANS action.
+
+      return Object.assign({}, state, {
+        books: []
+      });
+
```

Comment:
This is the only change that's not just renaming. 

I'm not sure why it wasn't a problem for borrow, but after I used the update book action to revoke a loan, Root would continue using the book data from the loans feed until the loans request completed, but the loading message would go away as soon as the revoke request was done. The loans request can take a while so it would look like your revoke didn't work.

A better way to handle this would be to have this reducer update the loan for that one book, and keep all the other loans. That way we wouldn't need to make another loans request at all. I created an issue for that here https://github.com/NYPL-Simplified/opds-web-client/issues/159.